src/en/anchira/src/eu/kanade/tachiyomi/extension/en/anchira/AnchiraHelper.kt

```diff
@@ -0,0 +1,33 @@
+package eu.kanade.tachiyomi.extension.en.anchira
+
+import kotlinx.serialization.decodeFromString
+import kotlinx.serialization.json.Json
+import okhttp3.ResponseBody
+import okio.ByteString.Companion.decodeBase64
+
+object AnchiraHelper {
+    const val KEY = "fuck niggers and faggots, and death to all jews"
```

Comment:
Will do a full review on my laptop later, but I recommend obfuscating this key somehow (even a simple base64 encode would suffice) so the repo doesn't potentially get nuked over a hentai extension.